R/render-table.R

```diff
@@ -3,33 +3,103 @@
 #' Creates a reactive table that is suitable for assigning to an \code{output}
 #' slot.
 #'
-#' The corresponding HTML output tag should be \code{div} and have the CSS class
-#' name \code{shiny-html-output}.
+#' The corresponding HTML output tag should be \code{div} and have the CSS
+#' class name \code{shiny-html-output}.
 #'
 #' @param expr An expression that returns an R object that can be used with
 #'   \code{\link[xtable]{xtable}}.
-#' @param ... Arguments to be passed through to \code{\link[xtable]{xtable}} and
-#'   \code{\link[xtable]{print.xtable}}.
+#' @param striped,condensed,hover,bordered Logicals: if \code{TRUE}, apply
+#'   the corresponding Bootstrap table format to the output table.
+#' @param width A string with the width of the table, expressed in any valid
+#'   CSS unit of length.
```

Comment:
Numbers are valid here too, right? (Because it's being fed into `validateCssUnit`?)

I'd also add a couple of examples of valid units--see `plotOutput` for an example.